2315.count-asterisks.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/count-asterisks/
+ */
+
+/**
+ * @param {string} s
+ * @return {number}
+ */
+var countAsterisks = function(s) {
+    //split by |
```

Comment:
This comment is unnecessary because the operation is obvious from the code below.